scripts/manager_actor2.lua

```diff
@@ -221,7 +237,11 @@ function getSkillLevel(nodeChar, type, points)
 		end
 	end
 
-	return nLevel, skilltype, relativelevel;

```

Comment:
There seems to be a bug with FGU where this triplet was not always returning all values. This seems to work around that problem altogether.